test/Mono.Linker.Tests/TestCases/Dependencies/WarningSuppressionExpectations1.cs

```diff
@@ -0,0 +1,6 @@
+﻿using System.Diagnostics.CodeAnalysis;
+
+[assembly: UnconditionalSuppressMessage ("", "IL2006", Scope = "member", Target = "M:Mono.Linker.Tests.Cases.WarningSuppression.Warnings.NestedType.Warning3")]
```

Comment:
Why is the first parameter (the category) blank? This is a required parameter on the attribute, it doesn't feel correct for it to be blank.

For SuppressMessage, we have:

```C#
        [SuppressMessage("Microsoft.Performance", "CA1801:ReviewUnusedParameters", MessageId = "isChecked")]
        [SuppressMessage("Microsoft.Performance", "CA1804:RemoveUnusedLocals", MessageId = "fileIdentifier")]
```

from the docs:

https://docs.microsoft.com/en-us/dotnet/api/system.diagnostics.codeanalysis.suppressmessageattribute.category?view=netcore-3.1#System_Diagnostics_CodeAnalysis_SuppressMessageAttribute_Category

> The Category property describes the tool or tool analysis category for which a message suppression attribute applies.

Reply:
Is there any preference on what to put here? Would `ILLinkWarningSuppression` be good?

Reply:
My understanding is that it is the name of the tool the attribute applies to - so it's not a "warning suppression". What would be more fitting is just `ILlinker` - basically the text we prefix all our messages (including warnings) with.

Reply:
BTW: From a casing perspective this should be spelled `ILLinker` - also in MessageContainer.ToMSBuildString. Would also be nice to have it as a constant used from both places.